packages/apps/crm/src/i18n/en.json

```diff
@@ -107,5 +108,9 @@
   "Crm_SliceAction_FetchProspect": "fetch prospects",
   "Crm_SliceAction_FetchProspectById": "get prospect by id",
   "Crm_SliceAction_UpdateProspectScore": "update prospect score",
-  "Crm_SliceAction_UpdateProspect": "update prospect"
+  "Crm_SliceAction_UpdateProspect": "update prospect",
+  "Crm_SliceAction_CreateCatalog": "create catalog",
+  "Crm_NewCatalog": "New catalog",
+  "Crm_Image": "Image",
+  "Crm_PdfFile": "PDF file"
```

Comment:
Please move this above slice actions to keep file readable ...
The objective is to separate all the translations coming form slice actions from the rest of the application.
Same comment for FR translations